plumbing/cache/object_lru.go

```diff
@@ -24,6 +24,11 @@ func NewObjectLRU(maxSize FileSize) *ObjectLRU {
 	return &ObjectLRU{MaxSize: maxSize}
 }
 
+// NewObjectLRUDefault creates a new ObjectLRU with the default cache size.
+func NewObjectLRUDefault() *ObjectLRU {
```

Comment:
no tested function

Reply:
Now cache tests are done twice. One with the previous cache and another time with a cache created with `NewObjectLRUDefault`.